wpengine/lostromos
# Add ability to check template

# What Are We Doing Here

This allows you to now run lostromos check and pass in a template and a CR to get the resulting template back.

## How to Verify

1. Check out this PR
2. Run `go run main.go check --templates ./test-data/templates/ --cr ./test-data/cr_nemo.yml`
3. Verify output matched what you would expect given the input.
